packages/flutter/test/material/outlined_button_test.dart

```diff
@@ -0,0 +1,1216 @@
+    expect(material.elevation, 0.0);
+    expect(material.shadowColor, const Color(0xff000000));
+    expect(material.shape, RoundedRectangleBorder(
+      side: BorderSide(
+        width: 1,
+        color: colorScheme.onSurface.withOpacity(0.12),
+      ),
+      borderRadius: BorderRadius.circular(4.0),
+    ));
+    expect(material.textStyle.color, colorScheme.onSurface.withOpacity(0.38));
+    expect(material.textStyle.fontFamily, 'Roboto');
+    expect(material.textStyle.fontSize, 14);
+    expect(material.textStyle.fontWeight, FontWeight.w500);
+    expect(material.type, MaterialType.button);
+  });
+
+  testWidgets('Does OutlinedButton work with hover', (WidgetTester tester) async {
+    const Color hoverColor = Color(0xff001122);
+
+    Color getOverlayColor(Set<MaterialState> states) {
+      return states.contains(MaterialState.hovered) ? hoverColor : null;
+    }
+
+    await tester.pumpWidget(
+      Directionality(
+        textDirection: TextDirection.ltr,
+        child: OutlinedButton(
+          style: ButtonStyle(
+            overlayColor: MaterialStateProperty.resolveWith<Color>(getOverlayColor),
+          ),
+          onPressed: () { },
+          child: const Text('button'),
+        ),
+      ),
+    );
+
+    final TestGesture gesture = await tester.createGesture(kind: PointerDeviceKind.mouse);
+    await gesture.addPointer();
+    await gesture.moveTo(tester.getCenter(find.byType(OutlinedButton)));
+    await tester.pumpAndSettle();
+    final RenderObject inkFeatures = tester.allRenderObjects.firstWhere((RenderObject object) => object.runtimeType.toString() == '_RenderInkFeatures');
+    expect(inkFeatures, paints..rect(color: hoverColor));
+
+    gesture.removePointer();
+  });
+
+  testWidgets('Does OutlinedButton work with focus', (WidgetTester tester) async {
+    const Color focusColor = Color(0xff001122);
+
+    Color getOverlayColor(Set<MaterialState> states) {
+      return states.contains(MaterialState.focused) ? focusColor : null;
+    }
+
+    final FocusNode focusNode = FocusNode(debugLabel: 'OutlinedButton Node');
+    await tester.pumpWidget(
+      Directionality(
+        textDirection: TextDirection.ltr,
+        child: OutlinedButton(
+          style: ButtonStyle(
+            overlayColor: MaterialStateProperty.resolveWith<Color>(getOverlayColor),
+          ),
+          focusNode: focusNode,
+          onPressed: () { },
+          child: const Text('button'),
+        ),
+      ),
+    );
+
+    FocusManager.instance.highlightStrategy = FocusHighlightStrategy.alwaysTraditional;
+    focusNode.requestFocus();
+    await tester.pumpAndSettle();
+
+    final RenderObject inkFeatures = tester.allRenderObjects.firstWhere((RenderObject object) => object.runtimeType.toString() == '_RenderInkFeatures');
+    expect(inkFeatures, paints..rect(color: focusColor));
+  });
+
+  testWidgets('Does OutlinedButton work with autofocus', (WidgetTester tester) async {
```

Comment:
I didn't see a corresponding test like this in contained button tests

Reply:
Added one.